examples/next/source/features/recently-active-users-page/menu-link-component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Link from 'next/link'
+import { Feature } from '@paralleldrive/react-feature-toggles'
+import { RECENTLY_ACTIVE_USERS_PAGE } from './feature-toggles'
+
+const MenuLink = () => (
+  <Feature
+    name={RECENTLY_ACTIVE_USERS_PAGE}
+    inactiveComponent={() => null}
+    activeComponent={
+      () => 
+        <li>
+          {/*
+            Eventually we may add a function that applies currently activated
```

Comment:
Don't we have access to initial features from any component under the features provider in the component tree?

Reply:
We do, I haven't yet created an easy to use function to apply them to a given path. #174